build.sc

```diff
@@ -72,8 +72,14 @@ object morphir extends Cross[MorphirModule](ScalaVersions.all) {
   }
 }
 trait MorphirModule extends Cross.Module[String] { morphir =>
+  import DevMode._
   val workspaceDir = millbuild.build.millSourcePath
 
+  trait CrossPlatformSwitchable extends CrossPlatform {
```

Comment:
Rather than adding this new trait, you could add the behavior here: https://github.com/finos/morphir-scala/blob/main/mill-build/src/millbuild/crossplatform/CrossPlatform.scala#L12

Reply:
Did that